### ISSUE TITLE
`str` object has no attribute `get` error on metric upload

### ISSUE THREAD

**Issue by Jbithell** [DataDog/datadogpy#269]
After initialisation, following code returns error below. 
```
single_metric = {
    'metric': 'uploadSpeed',
    'type': 'gauge',
    'points': [speedData[0], speedData[3]]
}
api.Metric.send(metrics="uploadSpeed", **single_metric)
```
```
File "main.py", line 25, in sendData
	api.Metric.send(metrics="uploadSpeed", **single_metric)
File "venv\lib\site-packages\datadog\api\metrics.py", line 140, in send
	return super(Metric, cls).send(attach_host_name=True, **metrics_dict)
File "venv\lib\site-packages\datadog\api\resources.py", line 74, in send
	attach_host_name=attach_host_name)
File "venv\lib\site-packages\datadog\api\api_client.py", line 99, in submit
	if obj_params.get('host', "") == "":
		AttributeError: 'str' object has no attribute 'get'
```

**Comment by dmangot** [comment]
Same issue on OpenBSD 6.3

**Comment by dabcoder** [comment]
Apologies for the delay in updating this thread. We managed to reproduce this issue, and we're currently looking into it.

**Comment by dabcoder** [comment]
We've addressed this issue here: #463. Basically `metrics` is a list of dictionaries, so this why the above code snippet resulted in this error @Jbithell (cc @dmangot).
